.github/workflows/release.yaml

```diff
@@ -28,14 +34,12 @@ jobs:
           key: cache-${{ env.cache-version }}-go-${{ env.go-version }}-${{ hashFiles('v2/Makefile') }}
       - run: make setup
         if: steps.cache-tools.outputs.cache-hit != 'true'
-      - run: make image
       - name: Login to ghcr.io
         run: echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u $GITHUB_ACTOR --password-stdin
-      - name: Push to ghcr.io
+      - run: make build
+      - name: Build & Push to ghcr.io
         run: |
-          IMAGE_TAG=${GITHUB_REF#refs/tags/v} # Remove "v" prefix.
-          docker tag coil:dev ghcr.io/cybozu-go/coil:$IMAGE_TAG
-          docker push ghcr.io/cybozu-go/coil:$IMAGE_TAG
+          make image IMAGE_TAG=${GITHUB_REF#refs/tags/v} # Remove "v" prefix.
```

Comment:
Did you mean `make push`? This only builds and loads an image locally.
You might also want to take a look at https://github.com/docker/build-push-action